src/internal_rep.rs

```diff
@@ -3568,9 +3963,16 @@ mod tests {
             let mut renames = BTreeMap::new();
             renames.insert("old_name".to_string(), "new_name".to_string());
             let renamed_statement = statement.get_renamed_statement(&renames);
-            let sexp = Sexp::from(&renamed_statement);
-            assert!(sexp.to_string().contains("new_name"));
-            assert!(!sexp.to_string().contains("old_name"));
+            match Sexp::try_from(&renamed_statement) {
```

Comment:
It's cleaner to just do "Sexp::try_from(&renamed_statement).unwrap()".  Unwrap is just panic on Err, so it's functionally equivalent but more concise and readable.

The general principle is that we don't panic in real code, and therefore don't use unwrap because it might panic.  But in test code, panicking is just test failure, so unwrap is idiomatic there.